apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -1189,6 +1189,36 @@ const PositioningEndEvent = () => {
   );
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30439&commitId=6f78db3f49488267b859f73208f2b09037287a79&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.Target display none.chromium.png|0| Added|
|Positioning.Target display none.target display- none.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>